src/CMake/FindOSMesa.cmake

```diff
@@ -37,6 +37,9 @@
 #   Eric Brugger, Tue Feb 26 12:55:26 PST 2019
 #   Add logic to install libOSMesa in lib directory.
 #
+#   Kathleen Biagas, Tue Jan 31, 2023
+#   Remove check for OSMESA_SIZE, it is no longer used.
```

Comment:
Do you know...does this mean the whole off-screen window size limit is no longer an issue (e.g. its handled properly at run time now)? Or, only that we've lost any ability to control or interrogate it?

Reply:
I can't recall if we lost the ability to interrogate. I just noticed it wasn't being used, hasn't been used in a while and thought to remove the logic.

Reply:
I know we still experience off screen rendering problems for largeish windows (>4096) in some instances.